.github/workflows/dev-cd.yml

```
@@ -77,22 +77,29 @@ jobs:
           key: ${{ secrets.EC2_PRIVATE_KEY }}
           script: |
-            IMAGE=${{ secrets.DOCKERHUB_USERNAME }}/noonddu-was:${{ steps.vars.outputs.sha }}
+            IMAGE=${{ secrets.DOCKERHUB_USERNAME }}/nuntteo-was:${{ steps.vars.outputs.sha }}
 
             echo "Deploying image: $IMAGE"
 
             echo ${{ secrets.DOCKERHUB_TOKEN }} | docker login --username ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
 
+            # 호스트 로그 디렉터리 준비 (권한 이슈 있으면 sudo 사용)
+            mkdir -p /var/log/nuntteo-was || true
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)